interview-questions.md

```diff
@@ -1,43 +1,125 @@
+# ASSESSMENT 5: Interview Practice Questions
+
 Answer the following questions.
 
 First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-What is a gem?
+1. What is a gem?
+
 Your answer:
+    Gems are ruby packages/package manager that contains ruby programs and libraries. They're used to install and contain data.
 
 Researched answer:
+    Ruby gems are not magic. They are open source libraries that contains Ruby programs and libraries and allow programmers to access and use them in their own projects without explicitly inserting those codes.
+
+2. What does the acronym CRUD stand for?
 
-What does the acronym CRUD stand for?
 Your answer:
+    Create, Read, Update, Delete. 
```

Comment:
Make sure to expand on your initial answer.  This is supposed to mimic how you would respond in an interview, and while it is correctly defining CRUD, there is a lot more you can share with the interviewer that you have a deeper level of understanding of this concept.